### DISCUSSION TITLE
REQUESTING HELP - URGENT PRODUCTION ISSUE: 404 errors followed by 504 errors

### DISCUSSION THREAD

**Discussion by asbansal** [vercel/community#3333] [Help] [answered]
### Summary

Hi,
Suddenly at around 21st July'23, 1:30 PM UTC, our production deployment in the mentioned project started giving 404 errors. Whereas other deployments (development, preview) were working fine.

It affected ~50 domains linked to the project and hugely impacted our relationship with our clients.

Our efforts for tracing back to the issue:
1. All deployments were done atleast 5 days ago and no configuration was changed on vercel.
2. It was working fine on our local systems, so no issue with database and authentication.
3. Nothing was clear from the logs so we had to urgently enable monitoring (which was of no use)

Our temporary fix:
On an urgent basis we created two other projects and linked same github repo and it started working fine. We then moved the domains to these projects.

Now, after an hour or so, other deployments (development, preview) also started failing

deployment url: [jee-partners-batch1-d4rp556zk-stepfly.vercel.app](https://jee-partners-batch1-jqpiwcdek-stepfly.vercel.app/)

Now, any domain linked to our project is not working.

We are unable to identify the root cause which is very essential for us as we cannot afford to have it occurred again. Please help us in identifying the issue.

As it is our only source of earning, we are reluctant to continue using vercel as only source of hosting provider. We have already started exploring backup options, and it would be great anyone could also recommend a solution (inside or outside vercel) as a backup option please.

PLEASE HELP US!
Thank you for your time
Ankur

### Example

https://jee-partners-batch1-jqpiwcdek-stepfly.vercel.app

### Steps to Reproduce

<img width="941" alt="image" src="https://github.com/vercel/community/assets/48391146/2006b77c-5c32-4ed2-b9b9-24e279846028">

**Comment by amyegan** [comment]
Hey @asbansal. I don't have insight into what serverless functions are running on the homepage of the site, but it sounds like there may be a service timing out. You should be able to get a little more information from the [runtime logs](https://vercel.com/docs/concepts/observability/runtime-logs#). You could also log more details by wrapping code in [try/catch statements](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) to get more error information. 

I wasn't able to replicate this problem with my test project. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give us a better view of what is going wrong for your project.

**Reply by adiestel** [reply]
My application is effected by the same issue.  Seeing the same behavior.  Up until today all of the deployments were working fine.  After today all of our build / deployments started throwing the above 504 errors. 

We were able to get production running again by rolling back to the release prior to 24 hours which still works.  Now any builds we do in other branches begin to fail with the same 504 after they are rebuilt today.  I opened a case with support but haven't heard back yet.  I thought maybe this issue could be related to the various issues that were reported with build today on the vercel-status page but the issue still seems to be happening even after those issues were resolved.

**Comment by vcshivani** [comment]
I faced the same issue. Requesting the Vercel team to check this in detail and tell us why this happened and what could be the solution for this. It is now working fine for my application when I created a new project with the same Github branch.

**Comment by amyegan** [comment, marked as answer]
We opened an incident report for this issue. You can follow this status page for the latest updates: https://www.vercel-status.com/incidents/vz2hwjfj1gjx
